packages/insomnia-app/app/ui/components/panes/grpc-request-pane/__tests__/use-action-handlers.test.ts

```diff
@@ -0,0 +1,78 @@
+import { createBuilder } from '@develohpanda/fluent-builder';
+import { renderHook } from '@testing-library/react-hooks';
+import { ipcRenderer } from 'electron';
+import { mocked } from 'ts-jest/utils';
+
+import { GrpcRequestEventEnum } from '../../../../../common/ipc-events';
+import { grpcIpcMessageParamsSchema } from '../../../../../network/grpc/__schemas__/grpc-ipc-message-params-schema';
+import { grpcIpcRequestParamsSchema } from '../../../../../network/grpc/__schemas__/grpc-ipc-request-params-schema';
+import { GrpcMethodTypeEnum } from '../../../../../network/grpc/method';
+import { prepareGrpcMessage as _prepareGrpcMessage, prepareGrpcRequest as _prepareGrpcRequest } from '../../../../../network/grpc/prepare';
+import { GrpcDispatch } from '../../../../context/grpc';
+import { GrpcActionTypeEnum } from '../../../../context/grpc/grpc-actions';
+import useActionHandlers from '../use-action-handlers';
+
+jest.mock('electron');
+jest.mock('../../../../../network/grpc/prepare', () => {
+  const module = {
+    prepareGrpcMessage: jest.fn(),
+    prepareGrpcRequest: jest.fn(),
+  };
+
+  return module;
+});
```

Comment:
![Screenshot_20211210_101923](https://user-images.githubusercontent.com/15232461/145597484-165afe03-b352-4623-a60c-a9ef2a16062f.png)

Besides one (pictured on the right above) which seems to need to do it so it can type the object more ergonomically, this would be the only module where we don't just return an object literal.  It'd be good to favor consistency
```suggestion
jest.mock('../../../../../network/grpc/prepare', () => ({
  prepareGrpcMessage: jest.fn(),
  prepareGrpcRequest: jest.fn(),
}));
```